target/linux/apm821xx/image/Makefile

```diff
@@ -174,20 +177,20 @@ endef
 
 define Device/netgear_wndap620
   $(Device/netgear_wndap6x0)
-  DEVICE_TITLE := Netgear WNDAP620 - Premium Wireless-N
+  DEVICE_MODEL := WNDAP620 - Premium Wireless-N
```

Comment:
One might consider brackets here: `WNDAP620 (Premium Wireless-N)`
Same below.
But I'm not really convinced about it.

Reply:
Tricky. But brackets might be consistent with the current device_model strings where the text is not part of the model name. This seems to be case for this model:
* https://www.netgear.de/support/product/WNDAP620.aspx
* https://www.netgear.com/business/products/wireless/premium-wireless/WNDAP620.aspx

(Btw., there also exists the WNDAP620-100PES and WNDAP620-100NAS in the wild)

Reply:
This might bite back as an ALT_MODEL, too, but I would not insist on that here and prefer to keep the current version.

Reply:
Current meaning without or without brackets? (since I made the change to brackets a few minutes before your comment)

Reply:
With brackets, like I approved below. :-)